lessons/Early-Math/identity-property/langs/en-us/index.adoc

```diff
@@ -167,33 +167,104 @@ Students explore two functions - `scale` and `rotate` - each of which consumes a
+@teacher{If students struggle with this idea, revisiting the idea of a "function machine" can be helpful!}
+
+@ifnotslide{
+[.embedded, cols="^.^6a,^.^6a,^.^6a,^.^6a",stripes="none"]
+|===
 
-Let's say we enter @show{(code '(scale 3 (triangle 30 "solid" "purple")))}. We can expect @proglang to produce a 90-pixel solid purple triangle. The original image's size was _tripled_.
+| @show{(coe '(multiply-by-one m))}
+| @show{(coe '(divide-by-one m))}
+| @show{(coe '(add-zero m))}
+| @show{(coe '(subtract-zero m))}
 
-@lesson-instruction{
-Use @starter-file{scale-rotate} to experiment with the `scale` function and complete @printable-exercise{scale-rotate-identity.adoc} through question 5.
+|===
 }
 
 
+No matter what value we use for @math{m}, we will get that value back.
+
+@slidebreak
+
+
+@lesson-roleplay{
+Select four students - one to _act out_ each of the four functions above (`multiply-by-one`, `divide-by-one`, `add-zero`, and `subtract-zero`). Make it clear to the class what each function's name is. Emphasize that each function expects a Number, and will produce Number.
+
+Just as with any acting career, this one will begin with a rehearsal. When I say, "*multiply-by-one 24*", you say, "*24*" Let's try it.
+
+- Teacher: *multiply-by-one... 24*
+- Student: 24!
+- Teacher: *divide-by-one... 366*
+- Student: _366_
+- Teacher: *add-zero... 4*
+- Student: _4_
+- Teacher: *subtract-zero... 16*
+- Student: _16_
+
+That was a great rehearsal. You're ready for the stage! Now it's the class' turn to give you cues! Who's got an in put for our one of our actors?
+
+Go around the room soliciting expressions from students until it's clear that everyone could run this script in their sleep.
+}
+
 === Investigate
 
-According to the Identity Property, we can also add or subtract 0 from a number - and we get that same number back.
+We just talked about four different functions. Each one consumed a number and produced a number. But what about functions that consume images? Can the Identity Property apply to those?
+
+@QandA{
+@Q{What is something we can do to this dog (below) that will _transform_ it... but still result in the *exact same image*?}
+}
+
+@right{@image{images/dog.png, 150}}
+
+@teacher{
+Solicit student responses and record them on the board. If students do not volunteer answers, consider sharing one or two of the following possible responses: flip it vertically twice; flip it horizontally twice; rotate it 360 degrees clockwise; rotate it 360 counter-clockwise; scale it by 1; slide it some distance and then return it to its original position.
+
+Students will be working in @proglang soon - but we recommend keeping things unplugged for now! You might even print up an image of the dog - or any other image you choose - to display on the board at the front of the room. Invite students to the board to demonstrate their ideas by manipulating it.}
+
+@slidebreak
+
+
+@lesson-instruction{
+- Open @starter-file{identity-prop} and click “Run.”
```

Comment:
The starter files defines `hello`, which is a text image. `dog` is unbound. Maybe you need to hit publish again?

Overall I love the way you lay this out, and I'm really happy with your worksheet